### PR TITLE
Add ciweimao extension 

### PR DESCRIPTION
**Pull request by zsakvo**
## Description

New extension let you use features from https://www.ciweimao.com via their mobile api.
With this, you could read, search, buy and delete books in your account.

## Type of change

- New extension

### All Commands

all-commands

### List Books

list-books

### List Shelves

list-shelves

### Search Books

search-books

### Catalog

catalog

### Reading

reading

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @zsakvo 👋 

Thanks for you contribution 🔥 

For a start we need to change the texts in Actionpanels to English according to https://developers.raycast.com/basics/prepare-an-extension-for-store#localization-language - can you do that?

Request a new review when it's done and then I'm looking into it 🔥

**Comment by zsakvo** [comment]
Hi @pernielsentikaer

According to the regulations, I have translate all text to English.

Thanks for your revision.

File: extensions/ciweimao/package.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  "title": "Ciweimao Reader",
+  "description": "Use full features of www.ciweimao.com via their mobile api.",
+  "icon": "command-icon.png",
+  "author": "zsakvo",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "List Books",
+      "subtitle": "All Books",
```
Comment (pernielsentikaer) [inline extensions/ciweimao/package.json:13]:
```suggestion
      "subtitle": "Ciweimao",
```

File: extensions/ciweimao/package.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      "name": "shelf",
+      "title": "List Shelves",
+      "subtitle": "All Shelves",
+      "description": "Show all shelves of the current account",
+      "mode": "view"
+    },
+    {
+      "name": "chapter",
+      "title": "Continue Reading",
+      "subtitle": "Read Last",
```
Comment (pernielsentikaer) [inline extensions/ciweimao/package.json:27]:
```suggestion
      "subtitle": "Ciweimao",
```

File: extensions/ciweimao/package.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      "description": "Continue read the last book you read",
+      "mode": "view"
+    },
+    {
+      "name": "search",
+      "title": "Search Novel",
+      "subtitle": "Search Something",
```
Comment (pernielsentikaer) [inline extensions/ciweimao/package.json:34]:
```suggestion
      "subtitle": "Ciweimao",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @zsakvo 

Thanks for the update, a few comments and corrections:

- If wrong credentials is added, “Continue Reading” just give Chinese output
- If wrong credentials is added, “Search Novel” don't have graceful error handling like the others.

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by pernielsentikaer** [comment]
Hi @zsakvo 👋 

Do you need any help?

**Comment by pernielsentikaer** [comment]
Hi @zsakvo 👋 

I'll give you a few days more to respond before closing this PR (and we can reopen at any time) 

Let me know if I can do anything to help you 💯

**Comment by pernielsentikaer** [comment]
Hi @zsakvo 👋 

I'm closing this, we can reopen it when you're ready to continue the work 😄

**Pull request closed without merging**